ui/app/css/itcss/components/send.scss

```diff
@@ -653,6 +654,11 @@
       }
     }
 
+    &__caret {
```

Comment:
This classname doesn't follow BEM. It has two "elements". But... the other classes in these styles are invalid BEM names as well, so I guess this might as well be too for consistency. Oh well.